src/Microsoft.SqlTools.Hosting/Utility/Logger.cs

```diff
@@ -5,76 +5,152 @@
 
 using System;
 using System.Diagnostics;
+using System.Globalization;
```

Comment:
Do I need to re-review this? Is it the same as the previous version and can I just reapply same comments here?